Behavioral.Automation.Template/Behavioral.Automation.Template.Scenarios/Behavioral.Automation.Template.Scenarios.csproj

```diff
@@ -0,0 +1,36 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <Authors>Quantori Inc.</Authors>
+    <Description>Specflow scenarios for demonstration of Behavioral.Automation framework features and testing.</Description>
+    <Copyright>Quantori Inc.</Copyright>
+    <RepositoryUrl>https://github.com/quantori/Behavioral.Automation</RepositoryUrl>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.0.0" />
+    <PackageReference Include="Selenium.WebDriver.ChromeDriver" Version="96.0.4664.4500" />
+	  <PackageReference Include="SpecFlow" Version="3.9.40" />
+	  <PackageReference Include="SpecFlow.NUnit" Version="3.9.40" />
+	  <PackageReference Include="SpecFlow.Tools.MsBuild.Generation" Version="3.9.40" />
+    <PackageReference Include="NUnit3TestAdapter" Version="4.1.0" />
+    <PackageReference Include="Behavioral.Automation" Version="1.7.0" />
```

Comment:
Can be removed.

Reply:
Done, but i kept SpecFlow.NUnit and NUnit3TestAdapter, they are still needed for test runs. Without first one no tests are shown in Test Explorer, without second one it just not starts